smos-docs-site/pages/configuration-reference.markdown

```diff
@@ -0,0 +1,89 @@
+---
+title: Customisation Reference
+date: 2020-04-07
+---
+
+# Misc #
+
+contexts
+
+: ???
```

Comment:
These are a little bit complex to explain.
Let me get back to this a bit later.